pkg/proto/pbgo/mocks/core/api_mockgen.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: /workspaces/datadog-agent/pkg/proto/pbgo/core/api.pb.go
+// Source: /Users/jesse.gumz/go/src/github.com/DataDog/datadog-agent/pkg/proto/pbgo/core/api.pb.go
```

Comment:
Might want to revert this?

Reply:
Thanks cc @truthbk it seems like there might be something we want to fix in terms of how the scripting is working here.